reference/5.0/PowershellGet/Uninstall-Module.md

```diff
@@ -16,33 +16,49 @@ Uninstalls a module.
 ## SYNTAX
 
 ### NameParameterSet (Default)
+
 ```
 Uninstall-Module [-Name] <String[]> [-MinimumVersion <Version>] [-RequiredVersion <Version>]
  [-MaximumVersion <Version>] [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ### InputObject
+
 ```
 Uninstall-Module [-InputObject] <PSObject[]> [-Force] [-WhatIf] [-Confirm] [<CommonParameters>]
 ```
 
 ## DESCRIPTION
-The **Uninstall-Module** cmdlet uninstalls the specified module from the local computer.
-You cannot uninstall a module if it has other modules as dependencies.
+
+The `Uninstall-Module` cmdlet uninstalls a specified module from the local computer. You can't
+uninstall a module if it has other modules as dependencies.
 
 ## EXAMPLES
 
-### Example 1: Get a module and uninstall it
+### Example 1: Uninstall a module
```

Comment:
Why did you want to change the example to not use the pipeline? It is a good use case. Also, we don't need to call Get-InstalledModule if we are not using the pipeline.

I think we need one example that is just a simple uninstall module by name. and one example using the pipeline.

Reply:
My initial tests were incorrect because the pipeline didn't work, so I split the commands. I read your feedback and retested, and the pipeline tests were successful. I updated the examples.